packages/eds-core-react/src/components/Popover/Popover.stories.tsx

```diff
@@ -10,23 +10,37 @@ import {
   EdsProvider,
   Density,
 } from '../..'
-import { more_vertical } from '@equinor/eds-icons'
+import { more_vertical, close } from '@equinor/eds-icons'
 import { Meta, Story } from '@storybook/react'
 
-const { Title, Content } = Popover
+const { Title, Content, Header } = Popover
 
 const StoryCenter = styled.div({
   display: 'flex',
   justifyContent: 'center',
   margin: '10rem',
 })
 
+const StyledCloseButton = styled(Button)(({ theme }) => {
```

Comment:
Our goal is for users to use just `Button` inside the `Popver.Header`. Looking at the styling in `Popover.Header` I don't think you need this?

Reply:
Now the close button is larger has 48px width and height. With the previous styling had 32px, shall i change this in style inline?